### ISSUE TITLE
File extension consistency 

### ISSUE THREAD

**Issue by dualfade** [mazen160/secrets-patterns-db#22]
Is there any reason some have the extension yaml and others yml ?  Easier to all be the same when making remote requests for the files. 

Thank you.
Sincerely,

**Comment by mazen160** [comment]
Good catch, I have reviewed all files within the project, and renamed them from `*.yaml` to `*.yml` for consistency.

One point to add that these files that were not consistent resides on the `datasets`, which is made as playground reference for data used to generate the databases.

Commit: https://github.com/mazen160/secrets-patterns-db/commit/6957f3149ab6f9e6347cbc0fcacd58ddebdf81d6

**Comment by dualfade** [comment]
Excellent ! Will be great. This all works wonderful with jxscout as well. Ive been testing this Caido.
